docs/api-docs/storefront/widgets/widgets-overview.md

```diff
@@ -348,7 +348,7 @@ Widgets are rendered on the storefront as a data tag in the HTML.
 |--|--|
 | Widgets | Widgets are the units of content to be placed on specific pages in a Stencil theme. Each widget is comprised of a widget configuration and a widget template. *There is a limit of `10,000` widgets per store.* |
 | Widget templates | Widget templates are Handlebars-enabled HTML templates which define the widget’s structure on a page. These templates can include conditional logic as well as looping. *There is a limit of `1,000` total custom widget templates per store. This does not include templates pre-provided by BigCommerce.* |
```

Comment:
| Widget templates | Widget templates are Handlebars-enabled HTML templates that define the widget's structure on a page. These templates can include conditional logic as well as looping. *There is a limit of `1,000` total custom widget templates per store. This limit does not include templates pre-provided by BigCommerce.* |